### PR TITLE
Array storage

### PR DESCRIPTION
**Pull request by rijilr**

### CODE REVIEW

File: student.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
+storage = window.localStorage;
+window.onload = createArrays();
```
Comment (Anjanavr) [inline student.js]: createArrays is also invoked from the head tag. Why is that?

File: student.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+function sortItem(item,array,table){
+    let node = JSON.parse(storage.getItem(array));
+    let sorted_array = node.sort(function(a,b){
+                                            if(isNaN(a[item])){
+                                                var x = a[item];
+                                                var y = b[item];
+                                                var x = x.toLowerCase();
+                                                var y = y.toLowerCase();
+                                                if (x < y) {return -1;}
+                                                if (x > y) {return 1;}
```
Comment (Anjanavr) [inline student.js]: Optimize this code segment.

File: student.js (hunk 1 above)
Comment (rijilr) [reply student.js]: resolved

File: student.js (hunk 2 above)
Comment (rijilr) [reply student.js]: resolved

**Pull request merged**